stl/inc/__msvc_string_view.hpp

```diff
@@ -221,7 +221,15 @@ struct _WChar_traits : private _Char_traits<_Elem, unsigned short> {
         // compare [_First1, _First1 + _Count) with [_First2, ...)
 #if _HAS_CXX17
         if constexpr (is_same_v<_Elem, wchar_t>) {
+#if _HAS_CXX20
+            if (_STD _Is_constant_evaluated()) {

```

Comment:
When `_HAS_CXX20` is `1`, we should probably use `_STD is_constant_evaluated()` (the standard version). Ditto below.